include/nn/swkbd/swkbd.h

```diff
@@ -0,0 +1,175 @@
+#pragma once
+
+#include <nn/types.h>
+
+namespace nn {
+namespace swkbd {
+enum Preset { Default, Password, UserName, DownloadCode, Max_Preset };
```

Comment:
These should be enum classes (or the enumerator names should be prefixed with the enum name, e.g. Preset_Default, KeyboardMode_LanguageSet1, etc.